jdaviz/configs/imviz/plugins/links_control/links_control.vue

```diff
@@ -1,25 +1,36 @@
 <template>
   <j-tray-plugin
-    description='Re-link images by WCS or pixels.'
+    description="Re-link images by WCS or pixels, or the rotate viewer."

```

Comment:
Definitely at least should be "rotate the viewer", but maybe we want to be consistent with other uses of "orientation"?

```suggestion
    description="Re-link images by WCS or pixels, or change the orientation."

```